src/components/atoms/Logo.tsx

```diff
@@ -1,10 +1,11 @@
 import React, { FC } from 'react';
 
 interface LogoProps {
-	src: string;
-	alt: string;
+  src: string;
+  alt: string;
   height: string;
+  onClick?: any;
```

Comment:
Please add a function type to `onClick`.
something like:
`onClick?: () => any`